statesman/single-page-project
# Unmedlify

The purpose of this pull request is to remove a couple of dependencies from Medley wraps, mainly with the favicon and apple-touch-icon.png files.

In doing so, I make a couple of other changes to the $meta array a the top of the file:
- I moved the `url` node to the top. The value remains the published URL of the project.
- I added `assets_path`, which would probably rarely change, but could if we need to adjust where our assets came from.
- I updated the `thumbnail`, `shortcut_icon` and `apple_touch_icon` values to just the name of the files
- In the meta data of the page, I combined the `url`, `assets_path` values to build the image urls.

To test:

Pull the branch and grunt the file and publish to the prod url. You can see [this example here](http://projects.statesman.com/news/single-page-project/).
